osu.Game/Database/BeatmapDifficulty.cs

```diff
@@ -10,7 +10,7 @@ public class BeatmapDifficulty
         /// <summary>
         /// The default value used for all difficulty settings except <see cref="SliderMultiplier"/> and <see cref="SliderTickRate"/>.
         /// </summary>
-        public const float DEFAULT_DIFFICULTY = 5;
+        public const float DEFAULT_DIFFICULTY = 0;

```

Comment:
This is the comment which github destroyed.

This change will break gameplay cursor auto sizing and beatmaps that don't have these values set either due to old beatmap versions that didn't have them in the first place, or because they were tampered with.